### PR TITLE
Project_documentation_WIFIcar

### PR DESCRIPTION
**Pull request by Teodora1807**
### Author

Signed-off-by: Teodora Dumitru <teodumitru18@gmail.com>

### CODE REVIEW

**Review by irina-b-dev — COMMENTED** [review]
Please rename "project_documentation" folder to your username (teodora.dumitru)

File: docs/project/2024/project_documentation/index.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (irina-b-dev) [inline docs/project/2024/project_documentation/index.md:75]: You will probably not need these libraries if you are not using ST7789 display

File: docs/project/2024/project_documentation/index.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+The robot will have a microcontroller board like Pico w which support Wi-Fi and is connected to a Wi-Fi module or module with the Wi-Fi function integrated, that allows the car to connect to a Wi-Fi network and receive data from a smartphone. I am going to use the pico w microcontroller, a motor driver module, 2 motors, 4 rechargeable batteries but only 2 used simutaneous,a battery charger and a battery holder, the skelet of the car(2 big wheels + a small one for the front,the platform) , jumper wires and also a switch so that the battery can last longer. I will also design a web interface to make possible the automatic movement of the car. For the assembly part i will use screws for the skeleton of the care and fludor for the components that need to be glued.
+More details for each component and the ispiration behind them can be seen on the Readme file on github.
+
```
Comment (irina-b-dev) [inline docs/project/2024/project_documentation/index.md:40]: Please also add a [block diagram](https://www.eleccircuit.com/understanding-electronics-block-diagrams-with-example/) for your project.

File: docs/project/2024/Teodora_Dumitru/index.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+# Wifi Robot Car
+
```
Comment (alexandruradovici) [inline docs/project/2024/Teodora_Dumitru/index.md:2]: Add a one line description of your project.

File: docs/project/2024/Teodora_Dumitru/index.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+## Motivation
+
+When I was little, besides dolls and legos , i liked remote controlled cars or even helicopters so this is more like an ambition for me.
+
+## Architecture 
+
+Add here the schematics with the architecture of your project. Make sure to include:
+ - what are the main components (architecture components, not hardware components)
+ - how they connect with each other
```
Comment (alexandruradovici) [inline docs/project/2024/Teodora_Dumitru/index.md:24]:
```suggestion
```

File: docs/project/2024/Teodora_Dumitru/index.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+| [2 wheels,1 small wheel, 2 motors, platform ] | The kit | [15 RON](second-hand) |
```
Comment (alexandruradovici) [inline docs/project/2024/Teodora_Dumitru/index.md:64]:
```suggestion
| [2 wheels,1 small wheel, 2 motors, platform ](link here) | The kit | [15 RON](link here) |
```
 If there is no link, delete the square brackets.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/Teodora_Dumitru/index.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
+# Wifi Robot Car
+
+I intend to do a robot car that can be controlled through a website accessed from the mobile phone or a computer . The website will have on the home page a controller with all 4 directions of movement.
```
Comment (alexandruradovici) [inline docs/project/2024/Teodora_Dumitru/index.md:3]:
```suggestion
A robot car that can be controlled through a website accessed from the mobile phone or a computer .
```

File: docs/project/2024/project_documentation/index.md
Comment (alexandruradovici) [inline docs/project/2024/project_documentation/index.md:1]: Delete this file

File: docs/project/2024/Teodora_Dumitru/SS-AM.png
Comment (alexandruradovici) [inline docs/project/2024/Teodora_Dumitru/SS-AM.png:1]: rename the folder to you curs.upb.ro username, that is the part in front of you @stud email.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please delete all the files and folders that you do not need.

**Comment by alexandruradovici** [comment]
Please fix the build issue, you have a broken link.

File: docs/project/2024/project_documentation/index.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+| [2 wheels,1 small wheel, 2 motors, platform ] | The kit | [15 RON](second-hand) |
+| [Battery Charger for 2] | To recharge the used batteries | [24,99 RON](https://www.optimusdigital.ro/ro/incarcatoare-de-baterii/11021-incarcator-1865026650-dublu-cu-cablu-usb-pentru-baterii-cu-litiu-ion.html) |
+| [4 x Batteries] | The power source of the whole project | [68 RON](https://www.dedeman.ro/ro/acumulator-li-ion-well-18650-3-7v-2200-mah/p/1050265) |
+| [Battery holder for 2] | ensure reliable power supply | [4,99 RON](https://www.optimusdigital.ro/ro/suporturi-de-baterii/941-suport-de-baterii-2-x-18650.html) 
+| [L298N Motor Driver Module] |  directions controlling  | [20 RON](https://www.optimusdigital.ro/ro/drivere-de-motoare-cu-perii/145-driver-de-motoare-dual-l298n.html)|
+| [Jumper wires]| To ensure the connections between components | [already have]|
```
Comment (alexandruradovici) [inline docs/project/2024/project_documentation/index.md:65]:
```suggestion
| Jumper wires| To ensure the connections between components | already have| 
```

File: docs/project/2024/project_documentation/index.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+| [2 wheels,1 small wheel, 2 motors, platform ] | The kit | [15 RON](second-hand) |
```
Comment (alexandruradovici) [inline docs/project/2024/project_documentation/index.md:60]:
```suggestion
| [2 wheels,1 small wheel, 2 motors, platform ] | The kit | [15 RON](second-hand) |
```

The *second-hand* generates the build the error.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/project_documentation/index.md
Comment (alexandruradovici) [reply docs/project/2024/project_documentation/index.md:1]: This is not solved.

Comment (alexandruradovici) [reply docs/project/2024/project_documentation/index.md:1]: Please delete this folder.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by alexandruradovici** [comment]
@Teodora1807 is this still in progress?

**Comment by Teodora1807** [comment]
I have uploaded it

**Comment by alexandruradovici** [comment]
Is there another pull request? This one needs changes.

**Comment by alexandruradovici** [comment]
Please delete the unused folder, keep only the folder with your username.

File: docs/project/2024/teodora.dumitru1807/index.md [hunk 9]
```diff
@@ -0,0 +1,122 @@
+# Wifi Robot Car
+
+A robot car that can be controlled through a website accessed from the mobile phone or a computer
+
+
+
+**Author**: Dumitru Teodora-Iulia \
+**GitHub Project Link**: [link_to_github](https://github.com/UPB-FILS-MA/project-Teodora1807.git)
+
+
```
Comment (alexandruradovici) [inline docs/project/2024/teodora.dumitru1807/index.md:10]:
```suggestion
:::info

**Author**: Dumitru Teodora-Iulia \
**GitHub Project Link**: [link_to_github](https://github.com/UPB-FILS-MA/project-Teodora1807.git)

:::
```

File: docs/project/2024/teodora.dumitru1807/index.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
+# Wifi Robot Car
+
+A robot car that can be controlled through a website accessed from the mobile phone or a computer
+
+:::info
+
+**Author**: Dumitru Teodora-Iulia \
+**GitHub Project Link**: [link_to_github](https://github.com/UPB-FILS-MA/project-Teodora1807.git)
```
Comment (alexandruradovici) [inline docs/project/2024/teodora.dumitru1807/index.md:8]:
```suggestion
**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-Teodora1807.git
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Pull request merged**